src/Google/Service/ShoppingContent.php

```diff
@@ -693,76 +692,6 @@ public function __construct(Google_Client $client)
           )
         )
     );
-    $this->orderpayments = new Google_Service_ShoppingContent_Resource_Orderpayments(
```

Comment:
We should check on this to... why is Orderpayments being removed wholesale?

Reply:
It looks like `Orderpayments` were removed between `2.0` and `2.1`